Dockerfile

```diff
@@ -15,6 +15,7 @@ RUN apt-get update \
     && rm -rf /var/lib/apt/lists/*
 
 ENV PUPPETEER_SKIP_CHROMIUM_DOWNLOAD true
+ENV CHROME_LOG_FILE /debug/chrome_debug.log
```

Comment:
can you try to pass this variable directly from `index.js`?

before calling microphoneBrowser:
```
process.env.CHROME_LOG_FILE = `/debug/${test_id}-microphone-chrome-debug.log`;
```

before calling listenOnlyBrowser:
```
process.env.CHROME_LOG_FILE = `/debug/${test_id}-listenonly-chrome-debug.log`;
```

Reply:
Sure !

Reply:
I had to do a bit different, because process.env.CHROME_LOG_FILE was being called with the same value for both tests. So i used 'env' param.

Reply:
cool!